mon-pix/app/utils/proposals-as-blocks.js

```diff
@@ -153,11 +159,10 @@ export default function proposalsAsBlocks(proposals) {
       const didAttachedLabel = block.attachLabel({ isInputField, isHiddenLabelNeeded, prevBlockText, questionIdx: inputCount });
       prevBlockText = didAttachedLabel ? '' : block.text;
 
-      if (isHiddenLabelNeeded || block.input) {
-        challengeResponseTemplate.addAsNewLine(block.get());
-      }
-      else if (j === parts.length - 1) {
-        challengeResponseTemplate.addAsNewLine(block.get());
+      if (isHiddenLabelNeeded
+          || block.input
+          || j === parts.length - 1) {
```

Comment:
Je me demandais qu'est-ce que ce "j" faisais là ... en remontant un peu je vois que c'est un iteratif pour la boucle for ... je me demande si cette boucle ne pourrait pas être sortie de la fonction pour que ça soit plus claire ? 
Toute cette fonction `proposalsAsBlocks` paraît un peu complexe et longue, je ne sais pas trop quoi proposer d'autre que sortir des petits bout et changer le nom de la variable j pour simplifier :/